workflow-templates/appstore-build-publish.yml

```diff
@@ -85,6 +85,33 @@ jobs:
           cd ${{ env.APP_NAME }}
           composer install --no-dev
 
+      - name: Check Cargo.toml
+        id: check_cargo
+        uses: andstor/file-existence-action@20b4d2e596410855db8f9ca21e96fbe18e12930b # v2
+        with:
+          files: "${{ env.APP_NAME }}/Cargo.toml"
+
+      - name: Install musl-tools
+        if: steps.check_cargo.outputs.files_exists == 'true'
+        run: |
+          sudo apt-get install musl-tools
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
What are we checking out this time around? 🤔

Reply:
copy paste error, fixed